alkos/kernel/arch/x86_64/interrupts/idt.cpp

```diff
@@ -0,0 +1,116 @@
+/* internal includes */
+#include <bit.hpp>
+#include <debug.hpp>
+#include <defines.hpp>
+#include <init.hpp>
+#include <kernel_assert.hpp>
+#include <panic.hpp>
+
+TODO_BY_THE_END_OF_MILESTONE0
+/**
+ * @todo Add various debug traces when snprintf available
+ */
+
+/* crucial defines */
+static constexpr u32 kIdtEntries  = 256;
+static constexpr u8 kDefaultFlags = 0x8E;
```

Comment:
maybe describe default flags used here in comment

Reply:
Sure great idea. To be honest I forgot to do it at the first place

Reply:
done